src/http_server.h

```diff
@@ -134,6 +134,21 @@ class HTTPMetricsServer : public HTTPServer {
       std::string address, int thread_cnt,
       std::unique_ptr<HTTPServer>* metrics_server);
 
+  // Used by tritonfrontend python bindings
+  // Unifying Create function called from tritonfrontend.h
+  // TraceManager, RestrictedFeatures, and SharedMemoryManager objects
+  // are not propagated or used by HTTPMetricsServer
+  static TRITONSERVER_Error* Create(
+      std::shared_ptr<TRITONSERVER_Server>& server,
+      const UnorderedMapType& options,
+      triton::server::TraceManager*
+          trace_manager,  // Not used HTTPMetricsServer
```

Comment:
Don't do that just to make a "unified" interface..

Reply:
Thanks for the suggestion! Removed unused args from `Create(...)` function.